container-core/src/test/java/com/yahoo/container/handler/metrics/PrometheusV1HandlerTest.java

```diff
@@ -109,6 +109,6 @@ void values_response_is_equal_to_test_file() {
     void consumer_is_propagated_to_metrics_proxy_api() {
         String response = testDriver.sendRequest(VALUES_URI + consumerQuery(CUSTOM_CONSUMER)).readAll();
 
-        assertTrue(response.contains(REPLACED_CPU_METRIC));
+        assertTrue(response.contains(REPLACED_CPU_METRIC), response);
```

Comment:
Test just randomly failed, and it would be nice to know why. Not related to this.